orc8r/gateway/python/magma/common/service_registry.py

```diff
@@ -85,6 +85,33 @@ def reset():
         """
         ServiceRegistry.get_registry()["services"] = {}
 
+    @staticmethod
+    def get_bare_bootstrap_rpc_channel(domain_name, port, rootca_path=None):
+        """
+        Return an RPC channel to bootstrap service in CLOUD, without referencing a
+        control proxy config.
+
+        Returns:
+            grpc channel
+        """
+        ip = 'bootstrapper-controller.%s' % domain_name
+
+        rootca_path = '/var/opt/magma/certs/rootCA.pem' if not rootca_path else rootca_path
+        with open(rootca_path, 'rb') as rootca_f:
+            rootca = rootca_f.read()
+            ssl_creds = grpc.ssl_channel_credentials(
+                root_certificates=rootca,
+            )
+
+        authority = 'bootstrapper-controller.%s' % domain_name
```

Comment:
is `authority` the same as `ip`?

Reply:
in this case, yes, will just use authority for uniformity